components/bl-signature-pad-component/dist/index.js

```diff
@@ -1,2 +1,2 @@
.onSaveClick()}},n.saveJPGButtonLabel||"Save as JPG")}function c(t){const{signaturePad:e,component:n,eventHandlers:i}=t;return React.createElement("button",{style:{display:"true"===n.savePNGButtonVisibility?"inline-block":"none"},onClick:()=>{e.isEmpty()?alert("Please provide a signature first."):r(e.toDataURL(),"signature.png"),i.onSaveClick&&i.onSaveClick()}},n.savePNGButtonLabel||"Save as PNG")}function l(t){const{signaturePad:e,component:n,eventHandlers:i}=t;return React.createElement("button",{style:{display:"true"===n.saveSignatureButtonVisibility?"inline-block":"none"},onClick:()=>{if(e.isEmpty())alert("Please provide a signature first.");else if(i.onSaveSignatureClick){const t=a(e.toDataURL());i.onSaveSignatureClick({signatureBlob:t})}}},n.saveSignatureButtonLabel||"Save Signature")}function d(t){const{signaturePad:e,component:n,eventHandlers:i}=t;return React.createElement("button",{style:{display:"true"===n.saveSVGButtonVisibility?"inline-block":"none"},onClick:()=>{e.isEmpty()?alert("Please provide a signature first."):r(e.toDataURL("image/svg+xml"),"signature.svg"),i.onSaveClick&&i.onSaveClick()}},n.saveSVGButtonLabel||"Save as SVG")}function u(t){const{signaturePad:e,component:n,eventHandlers:i}=t;return React.createElement("button",{onClick:()=>{const t=e.toData();t&&(t.pop(),e.fromData(t)),i.onUndoClick&&i.onUndoClick()}},n.undoButtonLabel||"Undo")}function v(t){const{signaturePad:e,component:n,eventHandlers:i}=t;return React.createElement("div",{className:"action-buttons-block"},React.createElement("div",{className:"action-change-buttons"},React.createElement(s,{signaturePad:e,component:n,eventHandlers:i}),React.createElement(o,{signaturePad:e,component:n,eventHandlers:i}),React.createElement(u,{signaturePad:e,component:n,eventHandlers:i})),React.createElement("div",{className:"action-save-buttons"},React.createElement(c,{signaturePad:e,component:n,eventHandlers:i}),React.createElement(h,{signaturePad:e,component:n,eventHandlers:i}),React.createElement(d,{signaturePad:e,component:n,eventHandlers:i}),React.createElement(l,{signaturePad:e,component:n,eventHandlers:i})))}function m({component:t,eventHandlers:n}){const i=React.useRef(null),o=React.useRef(null);function s(){const t=Math.max(window.devicePixelRatio||1,1),e=i.current;e.width=e.offsetWidth*t,e.height=e.offsetHeight*t,e.getContext("2d").scale(t,t),o.current.clear()}return React.useEffect((()=>{o.current=new(e())(i.current,{backgroundColor:"rgb(255, 255, 255)",penColor:t.penColor}),window.addEventListener("resize",s),s()}),[]),React.createElement("div",{className:"bl-customComponent-signature-pad"},React.createElement("div",{className:"wrapper"},React.createElement("canvas",{ref:i,onMouseOver:()=>n.onMouseOver(),onMouseOut:()=>n.onMouseOut()})),React.createElement("div",{className:"footer"},React.createElement("div",{className:"description"},t.description,t.description||"Sign Above"),React.createElement(v,{signaturePad:o.current,component:t,eventHandlers:n})))}})(),i})()));
```

Comment:
Looks like you didn't delete dist files after adding to gitignore

Try `git rm --cached path/to/file`